dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedTRSOpenApiIT.java

```diff
@@ -175,11 +176,16 @@ void testAggregatedMetricsPut() {
         extendedGa4GhApi.aggregatedMetricsPut(metrics, platform2, workflowId, workflowVersionId);
         workflow = workflowsApi.getPublishedWorkflow(workflow.getId(), "metrics");
         workflowVersion = workflow.getWorkflowVersions().stream().filter(v -> workflowVersionId.equals(v.getName())).findFirst().orElse(null);
+
         assertNotNull(workflowVersion);
         assertEquals(2, workflowVersion.getMetricsByPlatform().size(), "Version should have metrics for 2 platforms");
 
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform1));
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform2));
+        platform1Metrics = workflowVersion.getMetricsByPlatform().get(platform1);
+        Metrics platform2Metrics = workflowVersion.getMetricsByPlatform().get(platform2);
+
+        Map<String, Metrics> metricsGet = extendedGa4GhApi.aggregatedMetricsGet(workflowId, workflowVersionId);
+        assertEquals(metricsGet.get(platform1), platform1Metrics);
+        assertEquals(metricsGet.get(platform2), platform2Metrics);
```

Comment:
I think `platform1Metrics` and `platform2Metrics` should be on the left side since they're the expected values